string id #robot id, format is "psr-01", "psr-02", ...
bool reset #reset flag for PID controller
float32 theta_left_des #desired angular position for left wheel
float32 theta_right_des #desired angular position for right wheel
float32 omega_left_des #desired angular velocity for left wheel
float32 omega_right_des #desired angular velocity for right wheel
float32 alpha_left_des #desired angular acceleration for left wheel
float32 alpha_right_des #desired angular acceleration for right wheel
float32 duty_left_des #desired duty cycle for left wheel
float32 duty_right_des #desired duty cycle for right wheel

